src/Router.php

```diff
@@ -359,20 +359,20 @@ public static function run(EmitterInterface $emitter = null): void
             }
 
             if ($resolved === null) {
-                if (!self::$emitHttpExceptions) {
+                if (!is_subclass_of($e, HttpException::class)) {
```

Comment:
This is jank, just catch HttpExceptions seperately

Reply:
You mean just have [Catchers](https://github.com/tnapf/Router#catching)?

Reply:
```php
try {
    // ...
} catch (HttpException $e) {
    // ...
} catch (Exception $e) {
    // ...
}
```

Reply:
YOU CAN DO THAT!?!?